templates/firefox-ios.xliff

```diff
@@ -3864,9 +3864,9 @@ Private Tab</source>
         <source>Checking review quality</source>
         <note>Title for the card displayed when a shopping product has not been analysed yet but the analysis is in progress.</note>
       </trans-unit>
-      <trans-unit id="Shopping.OptInCard.DisclaimerText.Title.v120" xml:space="preserve">
-        <source>By selecting “Yes, Try It” you agree to the %1$@ by %2$@:</source>
-        <note>Text for the disclaimer that appears beneath the rating image of the Shopping Experience Opt In onboarding Card (Fakespot). The first parameter will be replaced by the Fakespot app name and the second parameter by the company name of Mozilla.</note>
+      <trans-unit id="Shopping.OptInCard.Disclaimer.Title.v120" xml:space="preserve">
+        <source>By selecting “Yes, Try It” you agree to the %@:</source>
```

Comment:
Looking at the current l10n comment for this string, it is going to read as `By selecting “Yes, Try It” you agree to the Fakespot:`
"the Fakespot" sounds wrong. Also, what comes after ":" - that should be called out in comments.

Shoudn't the placeholder rather be "privacy policy" or "terms of use"? Or is the l10n comment off? (either way, we should call out what comes after the colon

CC @ewachowiak for Content team feedback and @thatswinnie since she landed the string

Reply:
We have separate strings for "privacy policy" or "terms of use" as they are buttons.

Reply:
Ok, but what comes after the colon in this string? That should be called out in l10n comments.
Also "**the** Fakespot"?

Reply:
Just chatted with Katie from Content Design, and now I understand the whole picture.
Let's just update the l10n comment, stating that after the colon will come the strings calling out "privacy policy" and "terms of use", to avoid any possible localizer confusion. Thanks everyone!

Reply:
Merging this since we're already far in the cycle, and will merge again the updated comment on Monday once it's done.

Reply:
I have seen the screenshot, and now that I have the full picture, the string won't work with localizations. I'll merge this, send a note to localizers to work around this, and file a bug on iOS side. Thanks